WordPress/gutenberg
# FontSizePicker: Add `display: contents` rule to custom size select

Fixes: #68271 

## What, Why & How?
Fixes text overflow issues in the Font Size Picker component by implementing `display: contents` on `components-font-size-picker` element  for long font size names.

Changes:
Adds `display: contents` rule to fix text overflow
Adds a new stylesheet for the `font-size-picker` component.

## Testing Instructions
1. Add custom font sizes with long names in your theme’s theme.json file. Example:
```json
{
	"settings": {
		"typography": {
			"fontSizes": [
				{
					"slug": "extra-long-heading-one",
					"size": "16px",
					"name": "Extra Extra Extra Long Heading Style One"
				},
				{
					"slug": "extra-long-heading-two",
					"size": "20px",
					"name": "Extra Extra Extra Long Heading Style Two"
				}
			]
		}
	}
}
```
2. Navigate to the Site Editor (Appearance > Editor).
3. Open the Typography panel and try to select one of the custom font sizes with a long name.
4. Notice, that the overflow is prevented because of truncation.

## Screenshots

### Large Devices
|Before|After|
|-|-|
|![Screenshot 2024-12-25 at 11 34 51 AM](https://github.com/user-attachments/assets/429427dc-91f5-4c4b-a588-8937829f8651)|![Screenshot 2024-12-25 at 11 34 19 AM](https://github.com/user-attachments/assets/4c8936e1-2f67-4b07-8c32-f2c2199c00b4)|

### Mobile Devices
|Before|After|
|-|-|
|![Screenshot 2024-12-25 at 11 35 11 AM](https://github.com/user-attachments/assets/5ddb4cbe-aa1f-4a74-9b35-51fe708c64c6)|![Screenshot 2024-12-25 at 11 35 35 AM](https://github.com/user-attachments/assets/c57b42d8-fb12-4a2f-baa9-d839ea028315)|

## Screencast

https://github.com/user-attachments/assets/c943294e-1170-4db5-b649-331c0abeb285

The fix works regardless of the screen width.
